packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -1579,3 +1579,58 @@ export function updateAuthWithoutTrigger(cwd: string, settings: any): Promise<vo
       });
   });
 }
+
+export function updateAuthWithGroupTrigger(cwd: string, settings: any): Promise<void> {
+  return new Promise((resolve, reject) => {
+    spawn(getCLIPath(), ['update', 'auth'], { cwd, stripColors: true })
+      .wait('What do you want to do?')
+      .send(KEY_DOWN_ARROW)
+      .send(KEY_DOWN_ARROW)
+      .sendCarriageReturn()
+      .wait('Select the authentication/authorization services that you want to use:')
+      .sendCarriageReturn()
```

Comment:
Don't propagate the bad behavior we are having in existing tests to rely on option defaults, can lead to hard to debug errors, use explicit options and yes/no answers.

Reply:
same not needed